docs/en/week07/07.md

```diff
@@ -9,7 +9,7 @@ title: Week 7
 
 ## Lecture part B
 
-
+We discussed self-supervised learning, introduced training an Energy-based models, Latent Variable EBM, specifically K-means example. We also introduced Contrastive Methods including DAE, details of BERT, Contrastive Divergence.
```

Comment:
This stuff is still wrong! Uff!
```suggestion
We discussed self-supervised learning, introduced how to train an Energy-based models, discussed Latent Variable EBM, specifically with explained K-means example. We also introduced Contrastive Methods, explained DAE with topographic map, the training process, and how it can be used, followed by introduction to BERT. Finally, we talked about Contrastive Divergence and explained with topographic map as well.
```